src/multiple_choice/__init__.py

```diff
@@ -59,5 +63,16 @@ def delayedInit():
     if version.parse(mw.pm.profile['mc_conf'].get('version', 0)) < version.parse(default_conf_syncd['version']):
         updateLocalConfig()
 
-gui_hooks.profile_did_open.append(delayedInit)
 
+def update_multiple_choice_note_type_from_config(user_config: str):
+    """Set options according to saved user's meta.json in the addon's folder"""
+    user_config_dict = json.loads(user_config)
+    updateTemplate(mw.col, user_config_dict)
+    return user_config
+
+
+# Only execute addon after profile and collection are fully initialized
+gui_hooks.profile_did_open.append(manage_multiple_choice_note_type)
+
+gui_hooks.addon_config_editor_will_save_json.append(
+    update_multiple_choice_note_type_from_config)
```

Comment:
Move this to template.py

Reply:
I moved _everything_ to `template.py` now so `__init__.py` really is the entry point only. If we later want to get something out of `template.py` it's easier because you can `import` from `.template` but not from `__init__` (afaik).

As the addon is a `template` I think it's fitting to have the initializer there as well.